content/WebAuthn/WebAuthn_Developer_Guide/user-presence-vs-user-verification.adoc

```diff
@@ -0,0 +1,29 @@
+== User Presence vs User Verification ==
+
+This description is derived in part from the W3C's recommendation, https://www.w3.org/TR/webauthn/[Web Authentication: An API for accessing Public Key Credentials Level 1]. However, the distinction is also entirely applicable to YubiKeys outside the WebAuthn context.
+
+User **verification** serves to ensure through the presentation of a shared secret that the person authenticating to a service is in fact who they say they are for the purposes of that service. Or, in the words of the W3C recommendation, "...a user and the user’s client (containing at least one authenticator) work in concert to cryptographically prove to [that service, the RP] that the user controls the private key credential associated with a previously-registered public key credential". The user manipulates the authenticator that manages the key material in such a way as to invoke the ``authenticatorMakeCredential`` and ``authenticatorGetAssertion`` operations and the authenticator authorizes those operations locally. A shared secret can take various forms, such as password, PIN, fingerprint, public key credential, etc. The point is to distinguish one user from any other, i.e., uniquely identify the user.
```

Comment:
>User **verification** serves to ensure through the presentation of a shared secret [...]

There may be a secret shared between the user and authenticator (e.g., PIN), but let's not risk suggesting that the secret is shared with the RP. One of the key benefits of WebAuthn is that even multi-factor authentication doesn't require any shared secrets between the user/authenticator and the RP.

>The user manipulates the authenticator that manages the key material in such a way as to invoke the ``authenticatorMakeCredential`` and ``authenticatorGetAssertion`` operations and the authenticator authorizes those operations locally.

The `authenticatorMakeCredential` and `authenticatorGetAssertion` operations are always used regardless of if UV is; they take a parameter indicating whether UV is required.

```suggestion
User **verification** serves to ensure that the person authenticating to a service is in fact who they say they are for the purposes of that service. Or, in the words of the W3C recommendation, "...a user and the user’s client (containing at least one authenticator) work in concert to cryptographically prove to [that service, the RP] that the user controls the private key credential associated with a previously-registered public key credential". The RP directs the authenticator to perform user verification, the authenticator performs user verification locally and signals to the RP whether user verification was successful. User verification can take various forms, such as password, PIN, fingerprint, public key credential, etc. The point is to distinguish one user from any other, i.e., uniquely identify the user.
```